src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,24 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE_FOR_REGISTRATION = 18;
+    private static final int MIN_LIGHT_PASSWORD = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getPassword() == null) {
+            throw new NullPointerException("We can't have null values");
```

Comment:
```suggestion
            throw new RuntimeException("We can't have null values");
```
NPE is not the exception that you can throw
it should only appear when there was some action on null object